cluster/common.sh

```diff
@@ -474,7 +485,7 @@ function verify-kube-binaries() {
 # If KUBERNETES_SKIP_CONFIRM is set to y, we'll automatically download binaries
 # without prompting.
 function verify-release-tars() {
-  if ! $(find-release-tars); then
+  if ! find-release-tars; then
```

Comment:
is

Reply:
In ./cluster/common.sh line 490:
  if ! $(find-release-tars); then
       ^------------------^ SC2091: Remove surrounding $() to avoid executing output

Reply:
have we verified that this was not the intent of the original?